addons/mail/static/tests/chatter_tests.js

```diff
@@ -2516,8 +2516,8 @@ QUnit.test('followers widget: follow/unfollow, edit subtypes', async function (a
             if (route === '/web/dataset/call_kw/partner/read') {
                 nbReads++;
                 if (nbReads === 1) { // first read: should read all fields
-                    assert.strictEqual(args.args[1].length, 3,
-                        'should read "foo", "message_follower_ids" and "display_name"');
+                    assert.strictEqual(args.args[1].length, 4,
+                        'should read "foo", "message_follower_ids", "message_attachment_count" and "display_name"');
```

Comment:
I would replace this by 3 assertions `args.args[1].includes()` for the 3 items before.
This test is related to followers, and `message_attachment_count` should not be asserted here.